kodi_addon_checker/handle_files.py

```diff
@@ -49,7 +49,9 @@ def create_file_index(path: str):
         if ".git" in folders:
             folders.remove(".git")
         for file_name in files:
-            file_index.append({"path": root, "name": file_name})
+            if not os.path.basename(root).startswith("."):
```

Comment:
only ignores the files in hidden folders, not in non hidden sub-folders

- .test/tesfile: ignored
- .test/test/tesfile: **not** ignored

Reply:
- `.test/tesfile`
  - root: `[...]/.test`
  - file_name = `testfile`
  - basename(root): `.test`
  - ".test".startswith("."): `True`
  - -> ignored
- `.test/test/tesfile`
  - root: `[...]/.test/test`
  - file_name = `testfile`
  - basename(root): `test`
  - "test".startswith("."): `False`
  - -> **not** ignored
- `test/.tesfile`
  - root: `[...]/test`
  - file_name = `.testfile`
  - basename(root): `test`
  - "test".startswith("."): `False`
  - -> **not** ignored